components/section/Project.jsx

```diff
@@ -16,81 +16,148 @@ export default function Project() {
   ];
 
   const images = [
```

Comment:
You did a great Job, But for best practice, it is better if you put this data in a separate files to avoid a lot of lines of code in one file.